BDArmory/Settings/CompSettings.cs

```diff
@@ -0,0 +1,76 @@
+using System.Linq;
+using System.Reflection;
+using UnityEngine;
+
+using System.IO;
+using System.Collections.Generic;
+using System;
+using BDArmory.UI;
+
+namespace BDArmory.Settings
+{
+	public class CompSettings
+	{
+		// Settings overrides for AI/WM settings for competition rules compliance
+
+		static readonly string CompSettingsPath = Path.GetFullPath(Path.Combine(KSPUtil.ApplicationRootPath, "GameData/BDArmory/PluginData/Comp_settings.cfg"));
+		static public bool CompOverridesEnabled = false;
+		static readonly Dictionary<string, object> ComPOverrides = new()
+		{
+				// FIXME there's probably a few more things that could get set here for AI/WM override if needed in specific rounds.
+				//AI Min/max Alt?
+				{"PS_EXTEND_TIMEOUT", -1},
+				{"PS_EXTEND_DIST", -1},
+				{"PS_MONOCOCKPIT_VIEWRANGE", -1},
+				{"PS_DUALCOCKPIT_VIEWRANGE", -1},
+				{"PS_COCKPIT_FOV", -1},
+				{"PS_AVOID_THRESH", -1},
+				{"PS_AVOID_LA", -1},
+				{"PS_AVOID_STR", -1 },
+				{"PS_IDLE_SPEED", -1},
+				{"PS_DISABLE_SAS", false},
+		};
+
+        /// <summary>
+        /// Load RWP settings from file.
+        /// </summary>
+        public static void Load()
+        {
+            if (!File.Exists(CompSettingsPath))
+            {
+                Debug.LogError($"[BDArmory.CompSettings]: Override settings not present, skipping.");
```

Comment:
This shouldn't be a `Debug.LogError`. I think just a regular `Debug.Log` would suffice, maybe with a `DEBUG_OTHER` check too.

Maybe set `CompOverridesEnabled = false;` before returning here for the situation where someone deletes the file and hits "reload settings" without restarting KSP?